src/main/java/org/maxgamer/quickshop/command/subcommand/SubCommand_CleanGhost.java

```diff
@@ -107,14 +107,14 @@ public void onCommand(
                 // Shop exist check
                 Util.mainThreadRun(() -> {
                     Util.debugLog(
-                            "Posted to main server thread to continue accessing Bukkit API for shop "
+                            "->Posted to main server thread to continue accessing Bukkit API for shop "
                                     + shop);
                     if (!Util.canBeShop(shop.getLocation().getBlock())) {
                         MsgUtil.sendMessage(sender,
                                 ChatColor.YELLOW
                                         + "Deleting shop "
                                         + shop
-                                        + " because the target location is nolonger a shop or disallow create the shop.");
+                                        + " ->because the target location is nolonger a shop or disallow create the shop.");
```

Comment:
Untranlsable string that sends to server owner (shop deleted cause shop in disallowed region)